app/llm-demo/main.py

```
@@ -0,0 +1,132 @@
+            "role": "system",
+            "content": f"Answer using this context if relevant:\n{context}",
+        })
+    messages.append({"role": "user", "content": req.message})
+
+    try:
+        completion = client.chat.completions.create(
+            model=model,
+            messages=messages,
+            extra_headers={"x-request-id": request_id},
+        )
+    except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)